src/js/index.ts

```diff
@@ -12,47 +12,34 @@ export type {
 } from '@sentry/types';
 
 export {
-  addGlobalEventProcessor,
   addBreadcrumb,
   captureException,
   captureEvent,
   captureMessage,
-  getHubFromCarrier,
-  getCurrentHub,
-  Hub,
   Scope,
   setContext,
   setExtra,
   setExtras,
   setTag,
   setTags,
   setUser,
-  startTransaction,
-
-  // v8 spans
   startInactiveSpan,
   startSpan,
   startSpanManual,
   getActiveSpan,
   spanToJSON,
   spanIsSampled,
   setMeasurement,
-
-  // v8 scopes
   getCurrentScope,
   getGlobalScope,
   getIsolationScope,
   getClient,
   setCurrentClient,
   addEventProcessor,
-  metrics,
+  metricsDefault as metrics,
```

Comment:
shouldn't we also keep the name metricsDefault?

Reply:
Both JS and RN docs are referring to `Sentry.metrics`, that's why I renamed the export.

https://docs.sentry.io/platforms/javascript/metrics/#configure-metrics